src/components/ebay-menu-button/examples/18-typeahead/template.marko

```diff
@@ -0,0 +1,41 @@
+<h3>
+    Click the button, then type abcdefghijk to see focus change.
```

Comment:
I guess what I was going for was that they could type and sequentially go down the list. typing ab then abc would move the focus from ab to abc. But yeah if you don't like it we could change it.

Reply:
That's fine, lets just think of a better example. 
Maybe even Country, State, city, like
`United States`
`United States, California`
`United States, California, Los Angeles`
`United States, California, San Francisco`
`United States, California, San Jose`
Etc
Just something that came off the top of my head. We can do something else, maybe countries or such.

Ideally this would cover users usecases since the abc isn't a real usecase, but we can think of things that can coincide.

Reply:
Yeah, I think we should change it for something more real world